models/sales/hubspot__deal_stages.sql

```diff
@@ -26,6 +26,7 @@ with deals_enhanced as (
         deal_stage.deal_id || '-' || row_number() over(partition by deal_stage.deal_id order by deal_stage.date_entered asc) as deal_stage_id,
         deals_enhanced.deal_id,
         deals_enhanced.deal_name,
+        deals_enhanced.merged_deal_ids,
```

Comment:
Same comment about maybe adding a conditional if we add a variable

Reply:
added